features/conference/components/Conference.js

```diff
@@ -20,6 +25,7 @@ class Conference extends Component {
     constructor(props) {
         super(props);
         this.state = { isToolbarVisible: false };
+        this.toolbarTimeout = null;
```

Comment:
Generally, this constitutes the definition of a field and, consequently, our codying style requires jsdocs on it.